allennlp/data/dataset.py

```diff
@@ -102,12 +103,15 @@ def as_arrays(self,
 
         Returns
         -------
-        data_arrays : ``Dict[str, List[numpy.array]]``
+        data_arrays : ``Dict[str, Union[Dict[str, numpy.array], List[numpy.array]]``
```

Comment:
This type should match what's above: `Dict[str, Union[numpy.array, Dict[str, numpy.array]]`.